.github/workflows/release.yml

```diff
@@ -22,6 +22,9 @@ jobs:
       - name: Build package
         run: npm run build
 
+      - name: Test package
```

Comment:
we don't need execute tests in 2 pipelines in parallel, I think we can add a needs statement or something like that to the release workflow